nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -74,6 +81,24 @@ let
 
   drivesCmdLine = drives: concatStringsSep " " (imap1 driveCmdline drives);
 
+  # Creates a device name from a 1-based a numerical index, e.g.
+  # * `driveDeviceName 1` -> `/dev/vda`
+  # * `driveDeviceName 2` -> `/dev/vdb`
+  driveDeviceName = idx:
+    let letter = elemAt lowerChars (idx - 1);
+    in if cfg.qemu.diskInterface == "scsi" then
+      "/dev/sd${letter}"
+    else
+      "/dev/vd${letter}";
+
+  lookupDriveDeviceName = driveName: driveList:
+    (findSingle (drive: drive.name == driveName)
+      (throw "Drive ${driveName} not found")
+      (throw "Multilple drives named ${driveName}") driveList).device;
```

Comment:
`Multiple` typo pointed out by https://github.com/NixOS/nixpkgs/pull/92122#pullrequestreview-442590646

Reply:
I have force-pushed a fix to this into your PR (top commit).